### PR TITLE
Merge initial js-lib-redux-state-history

### PR DESCRIPTION
**Pull request by romasnavasinskas**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@paysera/error-message-resolver",
```
Comment (vbartusevicius) [inline package.json]: `error-message-resolver` ?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    "eslint-plugin-react": "^7.12.4",
+    "jest": "^24.1.0",
+    "jest-each": "^24.9.0",
+    "rimraf": "^2.6.3",
+    "webpack": "^4.29.6",
+    "webpack-cli": "^3.2.3",
+    "webpack-node-externals": "^1.7.2"
+  },
+  "dependencies": {
+    "redux-actions": "^2.6.5"
```
Comment (vbartusevicius) [inline package.json:60]: isn't `redux` aslo a dependency? - I see you're using `store.dispatch` somewhere

**Review by vbartusevicius — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (romasnavasinskas) [reply package.json]: Sorry, was using our another library as a guide/boilerplate to bootstrap common configurations that we usually use for javascript libraries. Copied common `package.json` configurations, this must have slipped through.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@paysera/js-lib-redux-state-history",
+  "version": "1.0.0",
+  "description": "A simple library that provides means to save Redux state and revert it when needed",
+  "main": "dist/main.js",
```
Comment (vbartusevicius) [inline package.json:5]: I don't see `dist/*` in this PR

**Review by vbartusevicius — CHANGES_REQUESTED** [review]

**Pull request merged**